crates/sui-types/src/full_checkpoint_content.rs

```diff
@@ -108,6 +163,13 @@ impl CheckpointTransaction {
         })
     }
 
+    pub fn removed_object_refs_post_version(&self) -> impl Iterator<Item = ObjectRef> {
+        let deleted = self.effects.deleted().into_iter();
+        let wrapped = self.effects.wrapped().into_iter();
+        let unwrapped_then_deleted = self.effects.unwrapped_then_deleted().into_iter();
```

Comment:
Do we need to care about unwrapped_then_deleted in this case?

Reply:
`unwrapped_then_deleted` does not make a diff to `latest_live_output_objects` while does make a diff to `eventually_removed_object_refs_post_version`, where the eventually removed object references will include versions from `unwrapped_then_deleted` instead of from the previous `wrapped`, this is semantically the right thing to me although I do not have a concrete case where this really matters.